test/model/tools/model_test/main.cpp

```diff
@@ -98,13 +99,27 @@ run_and_verify(std::shared_ptr<model::kv_workload> workload, const std::string &
         throw std::runtime_error("Failed to close file: " + workload_file);
 
     /* Run the workload in WiredTiger. */
+
+    std::vector<int> ret_wt;
     try {
-        workload->run_in_wiredtiger(home.c_str(), conn_config.c_str(), table_config.c_str());
+        ret_wt =
+          workload->run_in_wiredtiger(home.c_str(), conn_config.c_str(), table_config.c_str());
     } catch (std::exception &e) {
         throw std::runtime_error(
           "Failed to run the workload in WiredTiger: " + std::string(e.what()));
     }
 
+    /* Compare the return codes. */
+    size_t min_ret_length = std::min(ret_model.size(), ret_wt.size());
+    for (size_t i = 0; i < min_ret_length; i++)
+        if (ret_model[i] != ret_wt[i])
+            throw std::runtime_error("Return codes differ for operation " + std::to_string(i + 1) +
+              ": WiredTiger returned " + std::to_string(ret_wt[i]) + ", but " +
+              std::to_string(ret_model[i]) + " was expected.");
```

Comment:
1. I am unclear why return codes are compared in multiple places:
* test/common/util.cpp verify_workload()
* tools/model_test/main.cpp run_and_verify()
Is verify_workload() not used? If not, why not? If not, should verify_workload() be deleted?
I didn't take the time to figure this out. So do whatever is appropriate.
2. Perhaps make this message similar to the one from verify_workload() since they are the same thing. Similar but not the same since this has to tell which operation.

Reply:
The difference is that I only intended `verify_workload` to be used in the model's own unit tests, so it's more bare-bones, and any mismatch should result in an assert. (I was supposed to write that using `testutil_assert`, but I was not thinking). In contrast, `run_and_verify` is meant to be used for the actual testing of WT in Evergreen, so it has more polish and nice error handling.